.promu.yml

```diff
@@ -0,0 +1,9 @@
+go: 1.5.4
+repository:
+    path: github.com/prometheus/memcached_exporter
+build:
+    flags: -a -tags netgo
```

Comment:
no ldflags ?

We might want to replicate the `version` package of Prometheus and Alertmanager to exporters in the future.